mordant/src/jvmMain/resources/META-INF/native-image/com/github/ajalt/mordant/jni-config.json

```diff
@@ -0,0 +1,594 @@
+[
+  {
+    "name": "[Z"
```

Comment:
Sorry for the late question, but this entry looks odd, is it valid?

Reply:
I believe that's [a boolean array][1]. I don't know if it's necessary or not, but I don't thinks it's invalid.

[1]: https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-4.html#jvms-4.3.2